src/dotnet6/http-refit/AzFuncUni.Http/local-tests.http

```diff
@@ -0,0 +1,12 @@
+### Exercise 1 POST:
+POST http://localhost:7071/api/HelloWorldHttpTrigger1
+Content-Type: text/plain
+
+AzureFunctionsUniversity
+
+### Exercise 2 and 3 POSTs:
+POST http://localhost:7071/api/HelloWorldHttpTrigger2?name=World!
```

Comment:
Execution of HelloWorldHttpTrigger2 results in a HTTP 500 status since the interface is not registered. Can you fix that?

Reply:
That’s a good point.

I slightly refactored the code in `Program.cs` and updated the references from the corresponding CodeTour steps.